RELEASE.md

```diff
@@ -6,6 +6,20 @@
 - [GNU Make](https://www.gnu.org/software/make/) `v4.x`
 - [Kustomize](https://github.com/kubernetes-sigs/kustomize) `v1.3.x`
 
+## Github Workflow Test Matrix Checkup
+
+**For all releases**
+
+We maintain some integration tests with 3rd party components which we need to manually verify and update before cutting any release.
+
+- [ ] check the testing workflow (`.github/workflows/test.yaml`) and ensure that all matrix versions are up to date for Kubernetes releases. If there have been any new releases (major, minor or patch) of Kubernetes since the latest version seen in that configuration make sure the new versions get added before proceeding with the release.
+- [ ] check the release workflow (`.github/workflows/release.yaml`) and ensure that all matrix versions are up to date for Kubernetes releases. If there have been any new releases (major, minor or patch) of Kubernetes since the latest version seen in that configuration make sure the new versions get added before proceeding with the release.
+- [ ] check the release workflow (`.github/workflows/release.yaml`) and ensure that all matrix versions are up to date for [Istio][istio] releases. If there have been any new releases (major, minor or patch) of Istio since the latest version seen in that configuration make sure the new versions get added before proceeding with the release.
```

Comment:
Nobody will read the wall of text copy-pasted 3 times. And if one of these entries ever changes, the reader won't notice the difference.

Suggestion: create a high-level checkbox with 3 nested lower level checkboxes for individual files

Reply:
```suggestion
- [ ] check the testing workflow (`.github/workflows/test.yaml`) and ensure that all matrix versions are up to date for various component releases. If there have been any new releases (major, minor or patch) of those components since the latest version seen in that configuration make sure the new versions get added before proceeding with the release.
  - [ ] Kubernetes
  - [ ] Istio
```